### DISCUSSION TITLE
non-geospatial tile layer in PyDeck

### DISCUSSION THREAD

**Discussion by cornhundred** [visgl/deck.gl#5304] [Q&A] [answered]
Hi, is it possible to use a non-geospatial tile layer (e.g. https://deck.gl/examples/tile-layer-non-geospatial/) in PyDeck? 

I'm able to use data from a Pandas DataFrame to generate a BitMap layer (https://github.com/visgl/deck.gl/issues/5151), but it would be great to be able to load tiles from a much larger composite image.

**Comment by kylebarron** [comment, marked as answer]
I don't think the TileLayer is currently implemented in Pydeck because the `getTileData` prop is a JS function that can't currently be constructed in Python.

**Reply by cornhundred** [reply]
Thanks @kylebarron, I'll try something like this (https://github.com/visgl/deck.gl/issues/5151#issuecomment-730147020) where I can update the BitmapLayer image.

**Comment by cornhundred** [comment]
Would it be possible to have a flask server hosting the tile images (e.g. deepzoom JPEGs) and have a custom TileLayer in PyDeck make get requests to this flask server?
